core/drivers/crypto/crypto_api/acipher/rsa.c

```diff
@@ -102,7 +102,8 @@ TEE_Result crypto_acipher_rsanopad_decrypt(struct rsa_keypair *key,
 		CRYPTO_TRACE("Parameters error (key @%p)\n"
 			     "(msg @%p size %zu bytes)\n"
 			     "(cipher @0%p size %zu bytes)",
-			     key, msg, *msg_len, cipher, cipher_len);
+			     key, msg, msg_len != NULL ? *msg_len : 0,
```

Comment:
Good catch!
Prefer `msg_len ? *msg_len : 0` over `msg_len != NULL ? *msg_len : 0`.
Also, I think , when `msg_len` is NULL, size could be printed as a specific unexpected value, maybe `~0`.

This comment also applies to the other changes in this patch.